src/models/atmosphere/FGStandardAtmosphere.cpp

```diff
@@ -452,20 +465,29 @@ void FGStandardAtmosphere::ResetSLPressure()
 }
 
 //%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+// Calculates the density altitude given any temperature or pressure bias.
+// Currently the formula used is only valid up until the top of the Troposphere,
+// 11.019km - 36151.8ft.
+// https://en.wikipedia.org/wiki/Density_altitude
+// https://wahiduddin.net/calc/density_altitude.htm
```

Comment:
These comments should be moved to the header `FGStandardAtmosphere.h` so that they appear in the [project Doxygen documentation](https://jsbsim-team.github.io/jsbsim/classJSBSim_1_1FGStandardAtmosphere.html)

For Doxygen to parse correctly the comments, the syntax in `FGStandardAtmosphere.h` should be similar to
```c++
/** Calculates the density altitude given any temperature or pressure bias.
 Currently the formula used is only valid up until the top of the Troposphere,
 11.019km - 36151.8ft.
@see
 https://en.wikipedia.org/wiki/Density_altitude
 https://wahiduddin.net/calc/density_altitude.htm
*/
```